### ISSUE TITLE
Impossible de filtrer sur les tutos jamais publiés et les tutos en bêta dans les demandes d'aide

### ISSUE THREAD

**Issue by GerardPaligot** [zestedesavoir/zds-site#2877]
Il n'est pas possible de filtrer les tutoriels jamais publiés ni en bêta dans la liste des tutoriels qui ont besoin d'aide.

PS : Je marque évolution mais j'ai hésité avec bug.
PS 2 : C'est quoi un tutoriel jamais publié ?

**Comment by Situphen** [comment]
> C'est quoi un tutoriel jamais publié ?

Un tutoriel en écriture ?

**Comment by GerardPaligot** [comment]
Mais donc absolument tous les tutoriels privés des gens sont dans cette liste ?

**Comment by Situphen** [comment]
Non, seulement ceux qui ont au minimum une demande d'aide, et heureusement ! ;)

**Comment by Situphen** [comment]
J'ai raison @Eskimon ? :)

**Comment by GerardPaligot** [comment]
Si ça sert à ça, déjà il m'a fallu des explications pour comprendre donc d'autres pourraient se poser la même question. Mais surtout, je ne comprends pas du tout son utilité.

**Comment by GerardPaligot** [comment]
Ça doit pas être ça puisque mon tutoriel n'a jamais été publié mais il n'est pas marqué comme sélectionné.

**Comment by Situphen** [comment]
L'icône "Tutoriel jamais publié" n'est pas un filtre, c'est juste un indicateur, hein.

**Comment by GerardPaligot** [comment]
Je crois que tu ne comprends pas : J'ai un tutoriel en zone de bêta qui est donc la liste des tutoriels qui ont besoin d'aide. Si je comprends bien ce que tu m'affirme, l'indication qui signal que le tutoriel n'a jamais été publié devrait s'activer sur mon tutoriel mais ce n'est pas le cas. 

Puis, si c'est pas un filtre, faut le montrer. La, en tant que simple utilisateur qui n'a même pas suivi de loin le développement de cette fonctionnalité, tout ça n'est pas claire du tout. J'ai l'impression que toutes les indications sont des filtres mais que le système me permet de filtrer que sur 4 des 6 indications.

**Comment by Situphen** [comment]
> Je crois que tu ne comprends pas : J'ai un tutoriel en zone de bêta qui est donc la liste des tutoriels qui ont besoin d'aide. Si je comprends bien ce que tu m'affirme, l'indication qui signal que le tutoriel n'a jamais été publié devrait s'activer sur mon tutoriel mais ce n'est pas le cas. 

L'indication ne s'active que si le tutoriel est publié. Ce n'est pas le cas pour la majorité donc la plupart des indications de publications sont en mode OFF.

> Puis, si c'est pas un filtre, faut le montrer. La, en tant que simple utilisateur qui n'a même pas suivi de loin le développement de cette fonctionnalité, tout ça n'est pas claire du tout. J'ai l'impression que toutes les indications sont des filtres mais que le système me permet de filtrer que sur 4 des 6 indications. 

C'est vrai il n'y a que 4 filtres sur 6 indicateurs, mais est-ce que des filtres sur la publication et la mise en béta du tutoriel seraient pertinents ? Je ne pense pas. Qu'un tutoriel soit publié, en béta ou en rédaction ne change rien à la demande d'aide.

**Comment by GerardPaligot** [comment]
> L'indication ne s'active que si le tutoriel est publié. Ce n'est pas le cas pour la majorité donc la plupart des indications de publications sont en mode OFF.

Donc l'indication "Tutoriel jamais publié" s'active uniquement quand le tutoriel a déjà été publié. Soit c'est l'heure qui me fait dérailler, soit il y a un problème.

> C'est vrai il n'y a que 4 filtres sur 6 indicateurs, mais est-ce que des filtres sur la publication et la mise en béta du tutoriel seraient pertinents ? Je ne pense pas. Qu'un tutoriel soit publié, en béta ou en rédaction ne change rien à la demande d'aide.

Une mise en bêta, c'est une demande d'aide et c'est exactement le filtre que je voulais faire quand je me suis rendu sur la page. Ici, je donne un avis d'utilisateur final, je n'ai pas ma casquette du contributeur technique.

**Comment by Situphen** [comment]
> Donc l'indication "Tutoriel jamais publié" s'active uniquement quand le tutoriel a déjà été publié. Soit c'est l'heure qui me fait dérailler, soit il y a un problème.

Ça doit être l'heure ! :D
- Quand le tutoriel n'est pas publié, l'icône s'affiche en transparent avec le texte "Tutoriel jamais publié" au survol
- Quand le tutoriel est publié, l'icône s'affiche sans transparence avec le texte "Lire la version en ligne" au survol (et l'icône est un lien vers le tutoriel en ligne)

Si on prend une image provenant de [la page 4](https://zestedesavoir.com/tutoriels/aides/?page=4), le tutoriel au-dessus n'est pas publié et celui-ci en-dessous l'est !

![Voici l'image](http://i.gyazo.com/b2565dde656a5d9ddd4d5e7f88340668.png)

> Une mise en bêta, c'est une demande d'aide et c'est exactement le filtre que je voulais faire quand je me suis rendu sur la page. Ici, je donne un avis d'utilisateur final, je n'ai pas ma casquette du contributeur technique.

+1 Par contre je ne suis toujours pas convaincu du filtre sur la publication du tutoriel

**Comment by GerardPaligot** [comment]
Alors, il faut trouver un autre affichage pour cette indication "Tutoriel jamais publié" ou "Lire la version en ligne". Comme tu peux le voir, j'ai mis beaucoup de temps à savoir que ce n'était pas un filtre malgré le fait que rien ne le différentie des autres indications/filtres.

**Comment by firm1** [comment]
Je ne pense pas que ça soit efficace de discuter des évolutions de la zep03
sur github.

Il faudrait a mon avis en parler sur le forum pour que ce soit plus visible.

Le jeu. 2 juil. 2015 17:53, Gérard Paligot notifications@github.com a
écrit :

**Comment by GerardPaligot** [comment]
@firm1 Ta remarque soulève un problème plus chiant quant à la multiplication des endroits où poster ses bugs.

Mais en tant que utilisateur final, constatant un bug/évolution sur une page actuellement en prod, je viens le remonter sur GitHub, seul bugtracker officiel de la plateforme technique.

**Comment by firm1** [comment]
> @firm1 Ta remarque soulève un problème plus chiant quant à la multiplication des endroits où poster ses bugs.

C'est un problème connu de l'équipe de dev. Si tu veux en discutter [c'est par là](http://zestedesavoir.com/forums/sujet/1937/centraliser-la-gestion-des-bugs-dans-un-outil-unique/)

> Mais en tant que utilisateur final, constatant un bug/évolution sur une page actuellement en prod, je viens le remonter sur GitHub, seul bugtracker officiel de la plateforme technique.

Tout a fait, mais étant donné que c'est une évolution dont la solution n'est pas triviale, et donc qui demande l'avis de la communauté de zds, tu as plus de chance d'obtenir l'avis de la communauté sur le site en lui meme que sur github.
